Roots and Froots/Assets/Scripts/UI/Currency.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+public class Currency : MonoBehaviour
+{
+    private TextMeshProUGUI text;
+
+    private void Start() {
+        transform.GetComponent<TextMeshProUGUI>();
+        SetCurrencyValue(0);
+    }
+    public void SetCurrencyValue(int value) {
+        string curr_name;
+        if (value == 1) {
+            curr_name = "groat";
+        } else {
+            curr_name = "groats";
```

Comment:
nit: currency looks important can we have some tests?

Reply:
no. we need to push to prod now

Reply:
fair enough

Reply:
eat it then reet it mate, we don't need no tests